cryptol-remote-api/python/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Revision history for `cryptol` Python package
 
+## 2.12.0 -- 2021-11-19
+
+* v2.12 release
```

Comment:
Ideally, we'd include some details about the new features/bugfixes in this release. That's not your fault, of course—it's just slipped through the cracks as of late. In the future, I should remind PR submitters to include these sorts of details in the changelog so that we don't forget.

Would you feel up to summarizing some of the more notable changes that has happened since 2.11.7?

Reply:
It turns out your request is vacuously satisfied, as there have been no notable changes since 2.11.7! xD I did just push a few notes basically saying the 2.12 release is more for Cryptol-release syncing etc.

Reply:
Excellent. The best kinds of changes are vacuous ones.